core/build.gradle

```diff
@@ -55,22 +56,11 @@ dependencies {
 
     // Used by the 'relocateInternalDependencies', because ShadowTask doesn't let me specify a file.
     trimmedInternalDependency files(internalDependenciesJarTrimmed)
-
-    // Include the JAR generated by the 'relocateInternalDependencies' task into the class path
-    // so that the shaded classes can be referred by this project and other projects that depend on it.
-    compile files(internalDependenciesJarRelocated)
-}
-
-clean {
-    delete internalDependenciesJarTrimmed
-    delete internalDependenciesJarRelocated
 }
 
 task trimInternalDependencies(type: ProGuardTask) {
     injars files(configurations.internalDependency.collect()), filter: '!META-INF/**'
     outjars internalDependenciesJarTrimmed
-    libraryjars files(configurations.runtime.collect())
```

Comment:
Ah accidentally left this change in - I made it while experimenting with a different approach. But found the result seems to be the same without it. What do you think?